www/configs/settings.json

```diff
@@ -1,178 +1,4 @@
 {
-  "slide-layout": {
```

Comment:
The database is not installing on "fresh" installs. We read the hash and what not from the preferences file.